code/modules/food_and_drinks/recipes/food_mixtures.dm

```diff
@@ -40,7 +40,7 @@
 	var/resulting_reagent_purity
 
 /datum/chemical_reaction/food/pre_reaction_other_checks(datum/reagents/holder)
-	resulting_reagent_purity = holder.get_average_purity(/datum/reagent/consumable)
+	resulting_reagent_purity = holder.get_average_purity(/datum/reagent)
```

Comment:
```suggestion
	resulting_reagent_purity = holder.get_average_purity()
```

parameter is optional else it get's average of everything(every reagent is a subtype of /datum/reagent anyway)